src/va_list.cr

```diff
@@ -0,0 +1,20 @@
+require "c/stdarg"
+
+class VaList
+  # :nodoc:
+  getter to_unsafe
+
+  # :nodoc:
+  def initialize(@to_unsafe : LibC::VaList)
+  end
+
+  def self.open
+    ap = uninitialized LibC::VaList
+    Intrinsics.va_start pointerof(ap)
```

Comment:
Can you use `out ap` here?

Reply:
No because `va_start` signature needs `Void*` but `ap` is `VaList`.

Reply:
I see, makes sense.